layers/meta-xt-dom0-gen4/recipes-kernel/linux/linux-generic-armv8/defconfig

```diff
@@ -756,7 +756,7 @@ CONFIG_TRANSPARENT_HUGEPAGE_ALWAYS=y
 # CONFIG_TRANSPARENT_HUGEPAGE_MADVISE is not set
 # CONFIG_CLEANCACHE is not set
 # CONFIG_FRONTSWAP is not set
-CONFIG_CMA=y
+# CONFIG_CMA is not set
```

Comment:
Separate patch - hack/tmp to w/a binary size issue.

Reply:
Separate PR made and already merged.